tutor/src/helpers/plan.coffee

```diff
@@ -27,4 +27,9 @@ PlanHelper =
     {isPublishing, publishStatus}
 
 
+  isPlanOpen: (plan) ->
+    now = moment(TimeStore.getNow())
+    !! _.find plan?.tasking_plans, (tasking) -> moment(tasking.opens_at).isBefore(now)
```

Comment:
Yeah, I'd prefer to have all the components in the tree use the same logic.  I'll pass `isVisibleToStudents` into Footer and and then onto the child components.